data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,804 @@
+	"nettle capacity" 60
+	ammo `"Nettle" Kinetic Missile`
+	description "Nettle storage"
+
+
+outfit "Nettle Launcher"
+	category "Secondary Weapons"
+	licenses
+		"Twilight Guard"
+	cost 1370000
+	thumbnail "outfit/unknown"
+	"mass" 6
+	"outfit space" -15
+	"weapon capacity" -15
+	"gun ports" -1
+	"nettle capacity" 60
+	weapon
+		sprite "projectile/shardactive"
+			"no repeat"
+			"frame rate" .25
+		sound "finisher"
+		ammo `"Nettle" Kinetic Missile`
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"fire effect" "speckle fire"
+		"inaccuracy" 2.4
+		"velocity" 6
+		"lifetime" 15
+		"reload" 45
+		"firing energy" 6
+		"firing heat" 3
+		"infrared tracking" 0.9
+		"missile strength" 9
+		"submunition" "Nettle Active"
+	description "Nettle missiles are minimalist kinetic kill vehicles designed for use in situations where nuclear weapons are considered unnecessary. Little more than a payload of tungsten rods, a thruster package, and a nose mounted sensor array, they can be manufactured in bulk at minimal cost."
+	description "By splitting into a number of tungsten rods upon approaching a target, only a fraction of the payload actually impacts it, but the damage is also distributed over a large area of effect, making evasion difficult."
+	description "This is a standard modular launcher designed for use on light to medium warships. A coilgun launch system allows the missiles to be deployed with considerable initial velocity, making up for the limited fuel supply on the missile itself."
+
+
+outfit "Nettle Launch Array"
+	category "Secondary Weapons"
+	licenses
+		"Twilight Guard"
+	cost 1370000
+	thumbnail "outfit/unknown"
+	"mass" 9
+	"outfit space" -45
+	"weapon capacity" -45
+	"turret mounts" -1
+	"nettle capacity" 240
+	weapon
+		sprite "projectile/shardactive"
+			"no repeat"
+			"frame rate" .25
+		sound "finisher"
+		ammo `"Nettle" Kinetic Missile`
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"inaccuracy" 90
+		"velocity" 3
+		"lifetime" 30
+		"reload" 15
+		"burst reload" 9
+		"burst count" 12
+		"turret turn" 9
+		"firing energy" 3
+		"infrared tracking" 0.9
+		"missile strength" 9
+		"submunition" "Nettle Active"
+		"stream"
+	description "Nettle missiles are minimalist kinetic kill vehicles designed for use in situations where nuclear weapons are considered unnecessary. Little more than a payload of tungsten rods, a thruster package, and a nose mounted sensor array, they can be manufactured in bulk at minimal cost."
+	description "By splitting into a number of tungsten rods upon approaching a target, only a fraction of the payload actually impacts it, but the damage is also distributed over a large area of effect, making evasion difficult."
+	description "This large Vertical Launch Cell array is made up of forty cells, each with Nettles stacked six missiles deep. Designed for rapid vertical cold-launch, it needs an open space comparable to a turret mount to function properly."
+
+
+outfit "Nettle Active"
```

Comment:
Looking at the weapon itself:
That unusually long lifetime gives the missile an extremely long range - with .3 accel and .009 drag, it's travelling at a top speed of 33, giving it an actual range of 59,400 (the lower displayed ranges in-game are the result of using the launch velocity rather than the maximum speed of the projectile).
This is much, much more than anything else in the game - the longest range I'm aware of is the Firestorm, and its displayed 12540 range - more than the 9500 the projectile can actually travel, as its top speed is lower than the velocity it's launched with.
I think that this range is excessive. I'd suggest toning it down to be maybe 1/3, possibly 1/6 of what it is currently.
In terms of damage, the 180/270 shield/hull damage that this deals in total is quite average compared to the similarly-sized Meteor and Sidewinder, even before considering the high inaccuracy that adds a significant chance of some of the damage missing. So, the damage is fine.